src/api/services/TestDataService.ts

```diff
@@ -577,7 +577,7 @@ export class TestDataService {
         this.log.debug('bid.ListingItem.ListingItemTemplate.id:', bid.ListingItem.ListingItemTemplate.id);
         this.log.debug('listingItemTemplate.id:', listingItemTemplate.id);
         this.log.debug('listingItem.id:', listingItem.id);
-        this.log.debug('listingItem.seller:', listingItem.seller);
+        this.log.debug('listingItem.seller:', listingItem.Seller);
         this.log.debug('bid.bidder:', bid.bidder);
```

Comment:
should be listingItem.seller, only related models start with uppercase letters and seller is string iirc

Reply:
Fixed